### PR TITLE
Delete redundant backquotes

### PR DESCRIPTION
**Pull request by mefengl**
- remove extra space
- fix a typo

### CODE REVIEW

**Comment by mefengl** [comment]
image

fix my own typo

File: docs/installation/INSTALL_MAC.md [hunk 1]
```diff
@@ -45,11 +45,11 @@ First get the weights checkpoint download started - it's big:
...
-  brew install pyenv-virtualenv # you might have this from before, no problem
-  pyenv install anaconda3-2022.05
-  pyenv virtualenv anaconda3-2022.05
-  eval "$(pyenv init -)"
-  pyenv activate anaconda3-2022.05
+brew install pyenv-virtualenv # you might have this from before, no problem
+pyenv install anaconda3-2022.05
+pyenv virtualenv anaconda3-2022.05
+eval "$(pyenv init -)"
+pyenv activate anaconda3-2022.05
```
Comment (mefengl) [inline docs/installation/INSTALL_MAC.md:52]: extra space like this, removed

File: docs/installation/INSTALL_MAC.md [hunk 2]
```diff
@@ -189,7 +189,7 @@ There are several causes of these errors.
 - Third, if it says you're missing taming you need to rebuild your virtual
   environment.
 
-````bash
+```bash
 conda deactivate
 
 conda env remove -n ldm
```
Comment (mefengl) [inline docs/installation/INSTALL_MAC.md:195]: fix a typo

**Comment by mefengl** [comment]
@mh-dm it's so nice of you, noticed that it's `git pull --rebase origin development`, not `git pull --rebase upstream development`, why?

**Comment by mh-dm** [comment]
image
^ Commit message/description wasn't updated?

**Comment by mefengl** [comment]
> image
> 
> ^ Commit message/description wasn't updated?

QwQ ,I used amend option and then force push, it looks well locally. Next time I will double check.

**Pull request merged**